news/2018-07-20-the-qat-gazette-june-2018.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "The QAT Gazette: June 2018"
+date: 2018-07-20 10:00:00 +0000
+---
+
+A new month's worth of modding and mapping news is here for your convenience, with changes to Beatmap Nominator applications, Quality Assurance Team restructuring, and outcomes from the second public QAT meeting just some of this month's highlights. Read on inside to find out more!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+
+Hello everyone, [Protastic101](https://osu.ppy.sh/users/6712747) here, and it's that time of month again for modding and mapping news!
+
+As already mentioned in the headline, there's been a lot going on this past month which hopefully more than makes up for June's slow start. In addition to all that, we also have news on a new BN activity scoring system, a new draft of the osu!mania Ranking Criteria, and as usual, all the latest comings and goings within our teams.
+
+Before we get underway, a quick notice that the previously closed [osu! Modding Association](https://discord.gg/xChgWVd) Discord server is now open to the public. In addition to just being an all-around nice place to chat and chill about anything mapping or modding related, the server features a mapfeed tracker for all your nomination qualification needs, regularly hosted game nights, and of course, all the news updates you could ask for if you simply need to know what’s happening, as it's happening.
```

Comment:
`Discord server` should be linked.

Reply:
Should be fine either way since that's its official name after all, but I guess it doesn't hurt to have it this way